target/linux/ramips/image/mt7620.mk

```diff
@@ -244,6 +244,15 @@ define Device/u25awf-h1
 endef
 TARGET_DEVICES += u25awf-h1
 
+define Device/kimax_u35wf
+  DTS := U35WF
+  IMAGE_SIZE := 16064k
+  DEVICE_TITLE := Kimax U35WF
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ohci \
+  			kmod-scsi-core kmod-usb-storage
```

Comment:
Should be filesystem added as well? vfat/ext4?

Reply:
I will make one more change, excluding these packages since they are not needed for booting. I will commit back once tested again later. I am not with device right now, just in couple hours.